workflow/rocoto/workflow_tasks.py

```diff
@@ -1104,15 +1104,15 @@ def _get_ecengroups():
 
             return grp, dep, lst
 
-        eupd_cdump = 'gdas' if 'gdas' in self.app_config.eupd_cdumps else 'gfs'
+        #eupd_cdump = 'gdas' if 'gdas' in self.app_config.eupd_cdumps else 'gfs'
```

Comment:
Delete dead code.

Reply:
Just keep is as reference, and will delete dead code later.